src/components/ui/Notification/Notification.jsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect } from "react";
+import { toast, Toaster } from "react-hot-toast";
+import "./Notification.css";
+
+const Notification = ({ isCorrect, isTimeout, explanation, handleNextQuestion }) => {
```

Comment:
We are coupling the toast component with this specific scenerio. What would happen if I want to use this toast message in a different component?
Ideally you should composition pattern to solve this problem. Something like this:
```
<Toast>
<QuizNotication color={red} title={title} expanation={explanation}/>
<QuizButton handleClick={callback}>
Next Question
</QuizButton>
</Toast>
```
So we have seperatation of concern. We have a resuable toast component which can take any children it wants to make it resuable.